src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportStatusBauService.cs

```diff
@@ -48,11 +49,30 @@ public class ImportStatusBauService : IImportStatusBauService
 
                     return queryResults.Where(result =>
                     {
-                        if (result.Message == "")
+                        try
+                        {
+                            var m = JsonConvert.DeserializeObject<ImportMessage>(result.Message);
+
+                            if (m.Release?.Id == null || m.Release?.Publication?.Id == null || m.MetaFileName == null)
+                            {
+                                throw new NullReferenceException();
```

Comment:
I think you can just log the message and return false here, no need to throw an exception to immediately catch it again